config.c

```diff
@@ -287,6 +302,8 @@ bool apply_superset_style(
 			max(style->padding.bottom, target->padding.bottom);
 		target->padding.left = max(style->padding.left, target->padding.left);
 		target->border_size = max(style->border_size, target->border_size);
+		target->icons = style->max_icon_size || target->max_icon_size;
```

Comment:
I'm impressed no one noticed this gem of a copy-paste error. Oh well, better late than never.

Reply:
Eh, good catch